src/app/shared/models/providerStatus.model.ts

```diff
@@ -0,0 +1,6 @@
+import { Statuses } from '../enum/statuses';
+
+export interface ProviderStatus {
+  status: Statuses;
+  statusReason: string;
+}
```

Comment:
this interface is used only in one component for data initialization, I think we can provide it directly in the component. I don't see where we can possibly reuse this interface in the future 
![image](https://user-images.githubusercontent.com/27493959/208447769-3f80262b-a3aa-4096-8e6e-d0210906937d.png)
![image](https://user-images.githubusercontent.com/27493959/208447809-19a1542b-c1c7-4cb7-aca8-f625fcc9124f.png)